graphene_sqlalchemy/fields.py

```diff
@@ -94,18 +98,46 @@ def __init__(self, type, *args, **kwargs):
         super(SQLAlchemyConnectionField, self).__init__(type, *args, **kwargs)
 
 
-__connectionFactory = UnsortedSQLAlchemyConnectionField
+def default_connection_field_factory(relationship, registry):
+    model = relationship.mapper.entity
+    _type = registry.get_type_for_model(model)
+    return UnsortedSQLAlchemyConnectionField(_type._meta.connection)
+
+
+__connection_field_factory =  default_connection_field_factory
 
+def create_connection_field(relationship, registry):
+    return __connection_field_factory(relationship, registry)
 
-def createConnectionField(_type):
-    return __connectionFactory(_type)
+def register_connection_field_factory(connection_factory):
+    global __connection_field_factory
+    __connection_field_factory = connection_factory
 
+def unregister_connection_field_factory():
+    global __connection_field_factory
+    __connection_field_factory = default_connection_field_factory
+
+
+# TODO Remove in next major version
 
 def registerConnectionFieldFactory(factoryMethod):
-    global __connectionFactory
-    __connectionFactory = factoryMethod
+    log.warn(
+        'registerConnectionFieldFactory is deprecated and will be removed in the next '
+        'major version. Use register_connection_field_factory instead.'
```

Comment:
I created new functions using camelcase because it seems to be the pattern followed in this project.